### PR TITLE
Setup base files for the branch 8.0, forward port from 7.0 and makes pep8 pass

### PR DESCRIPTION
**Pull request by guewen**

### CODE REVIEW

**Comment by nbessi** [comment]
:+1:

File: .travis.yml [hunk 1]
```diff
@@ -1,48 +1,23 @@
...
-# Run tests with coverage
-# Only test modules in repo (list populated by directories in repo)
-# Preload modules before testing to only run tests of repo's modules
-# Include current directory and dependent repos in addons-path as well as official addons
 script:
-    - flake8 . --max-line-length=120 --exclude=__unported__ --filename=__init__.py --ignore=F401
-    - flake8 . --max-line-length=120 --exclude=__unported__,__init__.py
-    - odoo.py -d test --stop-after-init --init=$(python -c 'import os; print(",".join(x for x in os.listdir(".") if os.path.isdir(x) and not x.startswith(".") and x != "__unported__"))') --addons-path=$(pwd),`python -c "from distutils.sysconfig import get_python_lib; print(get_python_lib())"`/addons,$HOME/sale-workflow
-    - coverage run $(which odoo.py) -d test --test-enable --log-level=test --stop-after-init --init=$(python -c 'import os; print(",".join(x for x in os.listdir(".") if os.path.isdir(x) and not x.startswith(".") and x != "__unported__"))') --addons-path=$(pwd),`python -c "from distutils.sysconfig import get_python_lib; print(get_python_lib())"`/addons,$HOME/sale-workflow
+  - travis_run_flake8
```
Comment (lmignon) [inline .travis.yml:19]: travis_run_flake8 is no more required

**Comment by lmignon** [comment]
:+1:

File: .travis.yml [hunk 2]
```diff
@@ -1,48 +1,23 @@
...
 
-# Create databae
-# Pre-install modules and dependencies
-before_script: 
-    - createdb test
+install:
+  - git clone https://github.com/OCA/maintainer-quality-tools.git $HOME/maintainer-quality-tools
+  - export PATH=$HOME/maintainer-quality-tools/travis:$PATH
+  - travis_install_nightly
+  - git clone https://github.com/OCA/sale-workflow $HOME/sale-workflow -b 8.0
```
Comment (lmignon) [inline .travis.yml:16]: -b ${VERSION}

**Comment by yvaucher** [comment]
I close this as base is already setted

**Pull request closed without merging**